nxcomp/configure.ac

```diff
@@ -39,6 +35,10 @@ m4_ifdef([PKG_INSTALLDIR], [PKG_INSTALLDIR],
                        [],[with_pkgconfigdir='$(libdir)/pkgconfig'])
           AC_SUBST([pkgconfigdir], [${with_pkgconfigdir}])])
 
+PKG_CHECK_MODULES([JPEG], [libjpeg], [], [LIBJPEG_FALLBACK_CHECK])
+PKG_CHECK_MODULES(PNG, libpng)
```

Comment:
If touching these lines anyway, make that `PKG_CHECK_MODULES([...], [...])` too, please.